tasks/phpstan/fix-WPStubs-for-PHP-8_1.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// throw exception if anything fails
+set_error_handler(function ($severity, $message, $file, $line) {
+  throw new ErrorException($message, 0, $severity, $file, $line);
+});
+
+$file = 'vendor/php-stubs/wordpress-stubs/wordpress-stubs.php';
+
+$data = file_get_contents($file);
+
+$search_term = 'function readonly';
+
+if (! str_contains($data, $search_term)) {
```

Comment:
You are using a function only available in PHP 8, and we still haven't dropped PHP 7 support. Please use `strpos` or check PHP version before executing this code.

Reply:
Noted @wxa